data/json/recipes/basecamps/special_hardcoded.json

```diff
@@ -35,7 +35,7 @@
       { "id": "patrolling" }
     ],
     "blueprint_requires": [ { "id": "not_an_upgrade" } ],
-    "blueprint_name": "NPC-only.  If you see this it's a bug",
+    "blueprint_name": "NPC-only.  If you see this it's a bug.",
```

Comment:
This file wants adding here https://github.com/CleverRaven/Cataclysm-DDA/blob/be92113e11f0bcaf784b2bdd131fa064568909e7/lang/update_pot.sh#L45-L53 or `"//~": "NO_I18N"`ing